libs/dyn/dynvar/ref.go

```diff
@@ -8,8 +8,12 @@ import (
 )
 
 var (
+	// Regex should be in sync with _variable_regex in Python code.
+	//
+	// LINT.IfChange
```

Comment:
I like this. For the Python version, could you add a more explicit comment for maintainers that they should keep the two in sync?

Reply:
Makes sense, fixed

Reply:
What acts on these annotations?

Reply:
We don't have implementation in GitHub actions. Some open source projects, for instance, Chromium, implement it.  So this comment only captures intent. I wasn't able to find open source implementation

Reply:
Can you update it to some loud instruction instead?

The formatting makes me think it is taken care of automatically.